test/E2E/setupE2Etests.ps1

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+using namespace System.Runtime.InteropServices
+
+$IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
+
+$arch = [System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture.ToString().ToLowerInvariant()
+if ($IsWindowsEnv) {
+    $os = "win"
+    $FUNC_EXE_NAME = "func.exe"
+} elseif ($IsMacOS) {
+    $os = "osx"
+    $FUNC_EXE_NAME = "func"
+} else {
+    $os = "linux"
+    $FUNC_EXE_NAME = "func"
+}
+$FUNC_CLI_DOWNLOAD_URL = "https://functionsclibuilds.blob.core.windows.net/builds/2/latest/Azure.Functions.Cli.$os-$arch.zip"
+$FUNC_CLI_DIRECTORY = Join-Path $PSScriptRoot 'Azure.Functions.Cli'
+
+Write-Host 'Deleting Functions Core Tools if exists...'
+Remove-Item -Force "$FUNC_CLI_DIRECTORY.zip" -ErrorAction Ignore
+Remove-Item -Recurse -Force $FUNC_CLI_DIRECTORY -ErrorAction Ignore
+
+Write-Host 'Downloading Functions Core Tools...'
+Invoke-RestMethod -Uri 'https://functionsclibuilds.blob.core.windows.net/builds/2/latest/version.txt' -OutFile version.txt
```

Comment:
No reason to store to a file and then delete it:

```powershell
$version = Invoke-RestMethod ...
Write-Host "Using version: $version"
```

Reply:
GOOD Point